docs/DafnyRef/UserGuide.md

```diff
@@ -896,7 +896,7 @@ TO BE WRITTEN
 ### 25.8.4. Java
 
 The Dafny-to-Java compiler writes out the translated files of a file _A_`.dfy`
-to a directory _A_-java. The `-out` option can be used to choose a
+to a directory `_A_-java`. The `-out` option can be used to choose a
```

Comment:
```suggestion
to a directory _A_`-java`. The `-out` option can be used to choose a
```
Although I could also imagine removing the underscores and including the "A" in the verbatim text, too.

Reply:
The underscores are italics